paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerAdapterTest.kt

```diff
@@ -147,7 +147,7 @@ class CustomerAdapterTest {
             customerEphemeralKeyProvider = { error }
         )
         val result = adapter.retrievePaymentMethods()
-        assertThat((result.value as CustomerAdapter.Result.Failure).displayMessage)
+        assertThat((result.failureOrNull() as CustomerAdapter.Result.Failure).displayMessage)
```

Comment:
Nit (here and throughout the file)!

```suggestion
        assertThat(result.failureOrNull()?.displayMessage)
```